executor/runtime/docker/capabilities_test.go

```diff
@@ -38,3 +54,25 @@ func TestFuseProfile(t *testing.T) {
 	assert.Len(t, hostConfig.SecurityOpt, 2)
 	assert.Contains(t, hostConfig.SecurityOpt, "apparmor:docker_fuse")
 }
+
+func TestFuseProfile(t *testing.T) {
+	pod, conf, err := runtimeTypes.PodContainerTestArgs()
+	assert.NoError(t, err)
+	pod.Annotations[podCommon.AnnotationKeyPodFuseEnabled] = "true"
+	pod.Annotations[podCommon.AnnotationKeyPrefixAppArmor+"/"+pod.ObjectMeta.Name] = "docker_fuse"
```

Comment:
Yeah, I'm a little on the fence here.  The current behaviour is that we always use the apparmor profile set using `AnnotationKeyPrefixAppArmor`, regardless of whether Fuse is enabled or not.  The nice thing is that you can inspect the pod and determine what AA profile it's using.  The down side is that the TJC (or whatever's creating the pod) needs to know to set the AA profile to `docker_fuse` when enabling Fuse.

I could add fallback behaviour here: if no AA profile is specified and we request FUSE, use `docker_fuse` so that Fuse still works.  Thoughts?

Reply:
Yea I like that and think it is the safest. 
When we get to the point where *end users* are setting AA (hopefully never!) then we should rethink or do harder validation in TJC or the webhook or something.